spacy/tests/doc/test_span.py

```diff
@@ -680,3 +681,41 @@ def test_span_group_copy(doc):
     assert len(doc.spans["test"]) == 3
     # check that the copy spans were not modified and this is an isolated doc
     assert len(doc_copy.spans["test"]) == 2
+
+
+@pytest.mark.xfail
+@pytest.mark.parametrize("use_double_space", [False, True])
+def test_span_sentence_mapping(use_double_space: bool):
+    """Tests correct mapping of spans to sentences. This is currently failing due to some issue with the
+    span-to-sentence mapping.
+    use_double_space (bool): Whether to use double space after end of first sentence.
+    """
+    nlp = load("en_core_web_sm")
```

Comment:
The unit tests don't have access to any trained pipelines. You can export the output of `en_core_web_sm` and recreate the relevant docs by hand with `Doc` for the tests?

Reply:
Ah, right. Fixed in https://github.com/explosion/spaCy/pull/11910/commits/5349c1811c036b8740122ac96ac434fb83f667b4.